aosp_diff/preliminary/external/drm_hwcomposer/0006-fix-static-bound-to-dynamically-bound.patch

```diff
@@ -0,0 +1,216 @@
+From a3ba2d68322751d30502a12e4f28d8699f38a9fd Mon Sep 17 00:00:00 2001
+From: Xiao HaowenX <haowenx.xiao@intel.com>
+Date: Fri, 12 Nov 2021 10:51:51 +0800
+Subject: [PATCH] fix static bound to dynamically bound
+
```

Comment:
can give more comments in description, for example:
Previously connectors are bound to crtc in init stage, while connector counter and crtc counter are not matched, then caused some displays can't display when hotplug monitors into unbound connectors. Change to bind connector and crtc dynamically when detect connector changed to CONNECTED state in hotplug handling.